### PR TITLE
Restore outputing commits and constants in the pipeline.

### PR DESCRIPTION
**Pull request by lvella**

### CODE REVIEW

File: backend/src/estark/mod.rs [hunk 1]
```diff
@@ -131,45 +121,43 @@ impl<'a, F: FieldElement> EStarkFilesCommon<'a, F> {
...
-    fn write_files(
-        &self,
-        output_dir: &Path,
-        witness: &[(String, Vec<F>)],
-    ) -> Result<ProverInputFilePaths, Error> {
+    fn write_files(&self, output_dir: &Path) -> Result<ProverInputFilePaths, Error> {
+        let constants_name = if self.patched_constants.is_some() {
+            "constants_estark.bin"
+        } else {
+            "constants.bin"
```
Comment (leonardoalt) [inline backend/src/estark/mod.rs:147]: in this case, won't this have been written already?

File: pipeline/src/pipeline.rs [hunk 2]
```diff
@@ -920,6 +915,10 @@ impl<T: FieldElement> Pipeline<T> {
         self.output_dir.as_ref().map(|p| p.as_ref())
     }
 
+    pub fn force_overwrite(&self) -> bool {
```
Comment (leonardoalt) [inline pipeline/src/pipeline.rs:918]: this sounds like you're telling pipeline to force overwrite things. Maybe `is_force_overwrite`?

Comment (lvella) [reply pipeline/src/pipeline.rs]: fixed

**Comment by lvella** [comment]
Finally...

**Pull request merged**